cmd/manager/pause.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+var pauseCmd = &cobra.Command{
+	Use:   "pause",
+	Short: "A command that simply pauses",
+	Long:  `Pauses and waits for the SIGTERM signal`,
+	Run:   pause,
+}
+
+func init() {
+	rootCmd.AddCommand(pauseCmd)
+}
+
+func pause(cmd *cobra.Command, args []string) {
```

Comment:
Nit: Could we add a log message (or just a fmt.Println) that tells the user that the actual work is done in an init container? Otherwise `oc logs` doesn't show anything.

Reply:
will do